src/gl/canvas2dRe.re

```diff
@@ -0,0 +1,76 @@
+type t;
+
+/* Canvas API, following https://simon.html5.org/dump/html5-canvas-cheat-sheet.html */
+
+external save : unit = "" [@@bs.send.pipe: t];
+external restore : unit = "" [@@bs.send.pipe: t];
+
+/* Transformation */
+external scale : float => float => unit = "" [@@bs.send.pipe: t];
+external rotate : float => unit = "" [@@bs.send.pipe: t];
+external translate : float => float => unit = "" [@@bs.send.pipe: t];
+external transform : float => float => float => float => float => float => unit = "" [@@bs.send.pipe: t];
+external setTransform : float => float => float => float => float => float => unit = "" [@@bs.send.pipe: t];
```

Comment:
This looks like a prime use-case for labeled arguments. But the argument names proposed by MDN doesn't seem terribly helpful either:
```
external transform : m11::float => m12::float => m21::float => m22::float => dx::float => dy::float => unit = "" [@@bs.send.pipe: t];
```